src/view/pages/Farmer/index.tsx

```diff
@@ -2,47 +2,34 @@ import * as React from 'react';
 import createTabManager from '../../hoc/TabManager';
 import Collect from './tabs/Collect';
 import Loan from './tabs/Loan';
-import Buy from './tabs/Buy';
+// import Buy from './tabs/Buy';

```

Comment:
Remove the line since we won't be needing this.

Reply:
I left it as a comment because I changed it to Payments in a newer branch.